### PR TITLE
AT_126.001| New Item Page > Form Submission

### PR DESCRIPTION
**Pull request by marolespb8**

### CODE REVIEW

File: src/test/java/OkDisableTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+        String expectedError = "» This field cannot be empty, please enter a valid name";
+
+        getDriver().findElement(By.className("task-link-text")).click();
+        getDriver().findElement(By.cssSelector("div.btn-decorator")).click();
+        String actualError = getDriver().findElement(
+                By.cssSelector("#itemname-required.input-validation-message")).getText();
+
+        Assert.assertEquals(actualError, expectedError);
+
```
Comment (anonymoustuna) [inline src/test/java/OkDisableTest.java:18]: You might delete this empty line.

File: src/test/java/OkDisableTest.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class OkDisableTest extends BaseTest {
```
Comment (anonymoustuna) [inline src/test/java/OkDisableTest.java:6]: You might add space after this line.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/OkDisableTest.java (hunk 1 above)
Comment (marolespb8) [reply src/test/java/OkDisableTest.java:18]: Thank you.

File: src/test/java/OkDisableTest.java (hunk 2 above)
Comment (marolespb8) [reply src/test/java/OkDisableTest.java:6]: Thank you.

**Comment by zenicko** [comment]
Definitely qood! 
It would be better to delete empty lines.

**Comment by marolespb8** [comment]
Thank you for reviews.

**Pull request merged**